src/ai/backend/manager/models/agent.py

```diff
@@ -264,11 +264,15 @@ async def load_count(
         raw_status: str = None,
```

Comment:
If this function is capable of accepting `raw_status` value with `AgentStatus` type then we also need to update the type definition to reflect it.
```suggestion
        raw_status: Optional[str | AgentStatus] = None,
```